apps/full-site-editing/full-site-editing-plugin/global-styles/src/store.js

```diff
@@ -76,14 +77,17 @@ export default ( storeName, optionsPath ) => {
 			getOption( state, key ) {
 				return state ? state[ key ] : undefined;
 			},
+			getOptions( state ) {
+				return state ? state : undefined;
```

Comment:
minor: seems to be unnecessary a ternary there, but It's more readable. Maybe just returning the state? Anyway, it' up to you. Feel free to ignore this comment.

```suggestion
				return state;
```

Reply:
True, just did so to stay consistent with getOption. That said, there it serves a purpose, while here it doesn't. So I'd agree, let's remove it.